model/infinispan/src/main/java/org/keycloak/models/cache/infinispan/UserCacheSession.java

```diff
@@ -981,4 +976,13 @@ public List<AttributeMetadata> decorateUserProfile(String providerId, UserProfil
         }
         return List.of();
     }
+
+    private boolean isOrganizationDisabled(KeycloakSession session, UserModel delegate) {
```

Comment:
Can we have this method in `utils/Organizations` a remove it from here and `UserStorageManager`?

Reply:
We can but it would require moving that class to another package and increase the change set. Can you please open an issue so we can work as a follow-up, if you think it makes sense?